deluge/core/torrentmanager.py

```diff
@@ -111,10 +111,26 @@ def __init__(self,
         self.move_completed = move_completed
         self.move_completed_path = move_completed_path
 
+    def __eq__(self, other):
+        if not isinstance(other, TorrentState):
```

Comment:
Would prefer a `try..except` here. It is more likely to be an instance of TorrentState than not, and bit more pythonic.